### PR TITLE
Add meeting notes from Nov 11

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
@rossberg and @xtuc can you post links to your slides please?

File: 2019/CG-11-12.md [hunk 1]
```diff
@@ -44,3 +44,242 @@ Installation is required, see the calendar invite.
...
+SA: 0
+
+[Proposal passes.]
+
+
+### BigInt to stage 3
+SS: [presenting. TODO: Link]
+AR: agreed that there’s nothing to do in the reference interpreter for this proposal since it’s only about JS integration.
+
+TL: for the toolchain, emscripten currently does a transform that takes the i64s at the JS boundary and splits them into pairs of i32s. So for emscirpten we’d just want a flag to not do that.
```
Comment (xtuc) [inline 2019/CG-11-12.md]:
```suggestion
TL: for the toolchain, emscripten currently does a transform that takes the i64s at the JS boundary and splits them into pairs of i32s. So for emscripten we’d just want a flag to not do that.
```

File: 2019/CG-11-12.md [hunk 2]
```diff
@@ -44,3 +44,242 @@ Installation is required, see the calendar invite.
...
+F: 6
+N: 3
+A: 0
+SA: 0
+
+[Proposal passes.]
+
+
+### BigInt to stage 3
+SS: [presenting. TODO: Link]
```
Comment (xtuc) [inline 2019/CG-11-12.md]:
```suggestion
SS: [Link to slides](https://docs.google.com/presentation/d/1DJgaN-seUGuv123Lu6JvUwIudtAvpqZHCIFUE_Z85ew/edit?usp=sharing)
```

**Comment by dschuff** [comment]
I'll go ahead and merge this. It would still be nice to get a link from @rossberg

**Pull request merged**